candid/tests/serde.rs

```diff
@@ -116,8 +116,8 @@ fn test_reserved() {
 fn test_principal() {
     use candid::Principal;
     all_check(
-        Principal::from_text("ic:caffee59").unwrap(),
-        "4449444c0001680103caffee",
+        Principal::from_text("n7urw-bkejf-ceyaa-bnaaq-hsx75-y").unwrap(),
+        "4449444c000168010c4449444c0001680103caffee",
```

Comment:
The binary bytes doesn't look right. We may need to change the parser as well.

Reply:
Okay, this is actually correct, but too confusing as a test. I update to a simpler one.